
Taken from the Linux Tegra tree.  Patch has been accepted for 4.8-rc1

Origin: https://git.kernel.org/cgit/linux/kernel/git/tegra/linux.git/log/?h=for-4.8/arm64/dt


commit 5593eb76b602e3ab2d525cdeecd49c0978e3b0e6
Author: Thierry Reding <treding@nvidia.com>
Date:   Thu Jun 2 14:07:47 2016 +0200

    arm64: tegra: Enable debug serial on Jetson TX1
    
    Add a chosen node to the device tree that contains a stdout-path
    property which defines the debug serial port.
    
    Signed-off-by: Thierry Reding <treding@nvidia.com>

diff --git a/arch/arm64/boot/dts/nvidia/tegra210-p2180.dtsi b/arch/arm64/boot/dts/nvidia/tegra210-p2180.dtsi
index 8335bbe..5fda583 100644
--- a/arch/arm64/boot/dts/nvidia/tegra210-p2180.dtsi
+++ b/arch/arm64/boot/dts/nvidia/tegra210-p2180.dtsi
@@ -12,6 +12,10 @@
 		serial0 = &uarta;
 	};
 
+	chosen {
+		stdout-path = "serial0:115200n8";
+	};
+
 	memory {
 		device_type = "memory";
 		reg = <0x0 0x80000000 0x1 0x0>;
